.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  release:
+    types: [released]
+
+name: Upload Gutenberg plugin to WordPress.org plugin repo
+
+jobs:
+  upload:
+    name: Upload Gutenberg Plugin
+    runs-on: ubuntu-latest
+    if: github.event.release.assets[0]
+    env:
+      PLUGIN_REPO_URL: 'https://plugins.svn.wordpress.org/gutenberg'
+      STABLE_TAG_REGEX: 'Stable tag: [0-9]\+\.[0-9]\+\.[0-9]\+\s*'
+      SVN_USERNAME: ${{ secrets.svn_username }}
+      SVN_PASSWORD: ${{ secrets.svn_password }}
+      VERSION: ${{ github.event.release.name }}
+    steps:
+      - name: Check out Gutenberg trunk from WP.org plugin repo
+        run: svn checkout "$PLUGIN_REPO_URL/trunk"
+
+      - name: Get previous stable tag
+        id: get_previous_stable_tag
+        run: echo ::set-output name=stable_tag::$(grep "$STABLE_TAG_REGEX" ./trunk/readme.txt)
+
+      - name: Delete everything
+        working-directory: ./trunk
+        run: find . -maxdepth 1 -not -name ".svn" -not -name "." -not -name ".." -exec rm -rf {} +
+
+      - name: Download and unzip Gutenberg plugin asset into trunk folder
+        env:
+          PLUGIN_URL: ${{ github.event.release.assets[0].browser_download_url }}
+        run: |
+          curl -L -o gutenberg.zip $PLUGIN_URL
+          unzip gutenberg.zip -d trunk
+          rm gutenberg.zip
+
+      - name: Replace the stable tag placeholder with the existing stable tag on the SVN repository
+        env:
+          STABLE_TAG_PLACEHOLDER: 'Stable tag: [0-9]\+\.[0-9]\+\.[0-9]\+\s*'
+          STABLE_TAG: ${{ steps.get_previous_stable_tag.outputs.stable_tag }}
+        run: sed -i "s/${STABLE_TAG_PLACEHOLDER}/${STABLE_TAG}/g" ./trunk/readme.txt
+
+      - name: Commit the content changes
+        working-directory: ./trunk
+        run: |
+          svn st | grep '^?' | awk '{print $2}' | xargs svn add
+          svn st | grep '^!' | awk '{print $2}' | xargs svn rm
```

Comment:
Minor note — it's more for fun than anything — you can collapse those grep calls into awk:

```sh
svn st | awk '/^\?/ { print $2 }' | xargs …
```

Since, in practice, awk is most often used as a field selector, it may be more readable to leave this as is.

Reply:
:wave: a change along the lines of https://github.com/WordPress/gutenberg/pull/27886 is relevant for this I presume.

Reply:
Thanks for flagging, @nosolosw (and @youknowriad in https://github.com/WordPress/gutenberg/pull/27886#issuecomment-750847460). Added in 0564cc0ff6cf527fba4a92c9a5c4dff61f5748a6. (No OS check needed here since the action is guaranteed to run in a Linux container :tada: )